### PR TITLE
Schemaless tap

### PR DESCRIPTION
**Pull request by JingLin0**

### CODE REVIEW

File: tap_dynamics/discover.py [hunk 1]
```diff
@@ -25,10 +25,9 @@ def get_schema(odata_schema):
         json_type = "string"
         json_format = None
 
-        inclusion = "available"
+        inclusion = "automatic"
```
Comment (paddie) [inline tap_dynamics/discover.py:28]: 👏

File: tap_dynamics/__init__.py [hunk 2]
```diff
@@ -88,13 +88,11 @@ def main():
...
-        do_discover(service)
+        json.dump(catalog.to_dict(), sys.stdout, indent=2)
+
     else:
         sync(
-            service,
-            parsed_args.catalog,
-            parsed_args.state,
-            parsed_args.config["start_date"],
+            service, catalog, parsed_args.state, parsed_args.config["start_date"],
```
Comment (paddie) [inline tap_dynamics/__init__.py:97]: weird that this became one line though...

Comment (JingLin0) [reply tap_dynamics/__init__.py:97]: ` parsed_args.catalog` become `catalog`. it is shorter.

Comment (paddie) [reply tap_dynamics/__init__.py:97]: ahh, didn't catch that :D

**Pull request merged**